src/Watson/Sitemap/Sitemap.php

```diff
@@ -58,7 +60,7 @@ public function __construct(Cache $cache, Request $request)
      * Add new sitemap to the sitemaps index.
      *
      * @param  \Watson\Sitemap\Tags\Sitemap|string  $location
-     * @param  \DateTime|string  $lastModified
+     * @param  \DateTimeInterface|string|null  $lastModified
```

Comment:
We should use the DateTimeInterface over the specific implementation, this will allow Carbon as well